pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotRealtimeTableResource.java

```diff
@@ -449,6 +451,25 @@ public String getPauselessTableDebugInfo(
     }
   }
 
+
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  @Path("/tables/{tableName}/consumerWatermarks")
+  @Authorize(targetType = TargetType.TABLE, paramName = "tableName", action = Actions.Table.GET_IDEAL_STATE)
+  @ApiOperation(value = "Get table ideal state", notes = "Get table ideal state")
+  public WatermarkInductionResult inductConsumingWatermark(
+      @ApiParam(value = "Name of the realtime table", required = true) @PathParam("tableName") String tableName,
+      @Context HttpHeaders headers) {
+    try {
+      String table = DatabaseUtils.translateTableName(tableName, headers);
```

Comment:
Table copy won't change table name, is that correct?
That said, we cannot copy table in the same cluster (share controller) from one tenant to another, is that correct?